solidity/ecdsa/test/WalletRegistryGovernance.test.ts

```diff
@@ -2418,4 +2419,143 @@ describe("WalletRegistryGovernance", async () => {
       }
     )
   })
+
+  describe("beginNotifyOperatorInactivityGasOffsetUpdate", () => {
+    context("when the caller is not the owner", () => {
+      it("should revert", async () => {
+        await expect(
+          walletRegistryGovernance
+            .connect(thirdParty)
+            .beginNotifyOperatorInactivityGasOffsetUpdate(100)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when the caller is the owner and value is correct", () => {
```

Comment:
```suggestion
    context("when the caller is the owner", () => {
```

We do not really validate the value.

Reply:
Right, copy pasta from other tests.